src/filter/types.ts

```diff
@@ -60,4 +60,6 @@ export interface FilterItemCheckboxProps<T>
     labelExtractor?: ((item: T) => React.ReactNode) | undefined;
     /** Function to derive value from an item. If not set, checks `item.value`. */
     valueExtractor?: ((item: T) => string) | undefined;
+    /** Changes min-width to fit content */
+    useContentWidth?: boolean | undefined;
```

Comment:
prop name could be more targeted here

```suggestion
    /** Changes min-width of toggle in mobile view to fit content */
    useToggleContentWidth?: boolean | undefined;
```